Makefile.am

```diff
@@ -2,7 +2,7 @@ ACLOCAL_AMFLAGS = -I m4
 
 AM_CPPFLAGS = -I$(srcdir)/tsk
 AM_CFLAGS = -Wall -Wextra
-AM_CXXFLAGS = -Wall -Wextra -Woverloaded-virtual
+AM_CXXFLAGS = -Wall -Wextra
```

Comment:
`-Woverloaded-virtual` should not be removed here. We want warnings for this now that we've fixed all the old problems.

Reply:
not present now?